db-engine/src/main/java/com/splicemachine/db/impl/sql/compile/sqlgrammar.jj

```diff
@@ -18002,6 +18038,7 @@ reservedKeyword() :
 |    tok = <XMLSERIALIZE>
 |    tok = <XMLEXISTS>
 |    tok = <XMLQUERY>
+|    tok = <SPLIT_PART>
```

Comment:
Should that be a reserved keyword? I would rather have added it in the non reserved keyword list.

Reply:
done